app/views/sections/consignee/ConsigneeExciseView.scala.html

```diff
@@ -0,0 +1,74 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+
+@this(layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton,
+        h1: components.h1)
+
+@(form: Form[_], action: Call, enableDynamicContent: Boolean)(implicit request: DataRequest[_], messages: Messages)
+
+    @heading = {
+    @if(enableDynamicContent) {
+        @h1("consigneeExcise.temporaryConsignee.heading", Some(messages("consignee.subHeading")), "govuk-heading-l")
+    } else {
+        @h1("consigneeExcise.heading", Some(messages("consignee.subHeading")), "govuk-heading-l")
+    }
+    }
+
+    @layout(pageTitle = title(
+        form = form,
+        title = if(enableDynamicContent) {
+            messages("consigneeExcise.temporaryConsignee.title")
+        } else {
+            messages("consigneeExcise.title")
+        }
+    )) {
+
+        @formHelper(action, Symbol("autoComplete") -> "off") {
+
+            @if(form.errors.nonEmpty) {
+                @govukErrorSummary(ErrorSummaryViewModel(form))
+            }
+
+            @govukInput(
+                InputViewModel(
+                    field = form("value"),
+                    label = LabelViewModel(HtmlContent(heading))
+                ).withHint(
+                    Hint(
+                        id = Some("consignee-ern-hint"),
+                        classes = "govuk-hint",
+                        content = Text(
+                            if(enableDynamicContent) {
+                                messages("consigneeExcise.temporaryConsignee.hint")
+                            } else {
+                                messages("consigneeExcise.hint")
+                            }
+                        )
+                    )
+                )
+            )
+
+            @govukButton(
+                ButtonViewModel(messages("site.continue"))
+            )
```

Comment:
I think this should be `Save and continue` with an exit button. Check with Tom/Lexi.

Reply:
I've made inquiries into this